Makefile

```diff
@@ -8,5 +8,6 @@ clean:
 	true
 
 install:
+	rm -r $(DST)
 	mkdir -p $(DST)
 	cp -r templates/* -t $(DST)/
```

Comment:
Proč je tu potřeba `-t`? Měl jsem za to, že je potřeba jen tehdy, pokud `target` není posledním argumentem.

Reply:
Netuším, pravděpodobně pro bezpečnost, nicméně bez toho to bude fungovat úplně stejně.